app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -0,0 +1,379 @@
+import React from "react";
+import { forEach } from "lodash";
+import PropTypes from "prop-types";
+import ReactPanZoom from "@ajainarayanan/react-pan-zoom";
+
+import RemoveIcon from "~/components/ui/icons/RemoveIcon";
+import NetworkGraph from "~/components/visualizations/NetworkGraph.js";
+import cs from "./pipeline_viz.scss";
+
+const START_NODE_ID = -1;
+const END_NODE_ID = -2;
+
+class PipelineViz extends React.Component {
+  constructor(props) {
+    super(props);
+    this.pipelineVersion = this.props.stageResults.pipeline_version;
+    this.stageGraphsInfo = [];
+    this.graphs = [];
+
+    this.stageNames = [
+      "Host Filtering",
+      "GSNAPL/RAPSEARCH alignment",
+      "Post Processing",
+    ];
+    if (this.props.admin) {
+      this.stageNames.push("Experimental");
+    }
+
+    this.state = {
+      stagesOpened: [true, true, true, true],
+      zoom: 1,
+    };
+
+    this.handleMouseWheelZoom = this.handleMouseWheelZoom.bind(this);
```

Comment:
if you're using babel with the right plugins, you should be able to declare the `handleMouseWheelZoom` function as `handleMouseWheelZoom = () =>` in place outside of the constructor

Reply:
It is possible...